src/main/java/org/verdictdb/core/querying/ola/AsyncAggExecutionNode.java

```diff
@@ -72,6 +72,9 @@
 
   private Map<Integer, String> scrambledTableTierInfo;
 
+  // This is the Map that maps the aggregation alias to its column contents
```

Comment:
Please add more explanations with examples. Simply saying "column contents" does not provide much information.